app/models/forum.rb

```diff
@@ -0,0 +1,3 @@
+class Forum < ApplicationRecord
+  has_and_belongs_to_many :users
```

Comment:
I think you're doing a tutorial but just want to note that I don't see `has_and_belongs_to_many` very often in my work. The preferred way is with a "join" model, which I think you do in a different exercise (`has_many through:`)

Reply:
Got it and, yes, we went through `has_many :through`, though now I've already forgot the distinction between the two. Will review now,